
            Now there is a CVE in this repo:The is_safe_url function in utils/http.py in Django 1.4.x before 1.4.6, 1.5.x before 1.5.2, and 1.6 before beta 2 treats a URL\'s scheme as safe even if it is not HTTP or HTTPS, which might introduce cross-site scripting (XSS) or other vulnerabilities into Django applications that use this function, as demonstrated by "the login view in django.contrib.auth.views" and the javascript: scheme..
            Now the following functions is the cause for the CVE:django/contrib/auth/tests/test_views.py:test_security_check();django/contrib/auth/tests/test_views.py:test_security_check();django/utils/http.py:is_safe_url();django/utils/http.py:is_safe_url();
            